src/Common/Update.php

```diff
@@ -119,12 +119,13 @@ public function orWhere($cond)
      *
      * @param string $col The column name.
      *
-     * @return $this
+     * @param array  $value
```

Comment:
still not quite understanding: should `@param` have one blank line between each declaration, or should they be on consecutive lines with no blank line between them